controllers/multi_cluster_app_dispatcher.go

```diff
@@ -22,7 +22,6 @@ import (
 
 var multiClusterAppDispatcherTemplates = []string{
 	"mcad/configmap.yaml.tmpl",
-	"mcad/service.yaml.tmpl",
 	"mcad/serviceaccount.yaml.tmpl",
 	"mcad/deployment.yaml.tmpl",
```

Comment:
I think the corresponding ports from the Deployment can also be removed:

```yaml
ports:
  - name: https
     containerPort: 6443
     protocol: TCP
  - name: http
     containerPort: 8080
    protocol: TCP
```

Reply:
Great catch! Changes were made now. 

I do have a few thoughts though. Should we also be removing items from the [MCAD test data](https://github.com/project-codeflare/codeflare-operator/tree/5209f7a7cea17b303127bd5132ec9e02a476ab84/controllers/testdata) ?  

I.e., in mcad_test_results, there are some rolebindings and the same mentioned ports in the deployment of these tests.

These tests seem to run on the `tag-and-build` workflow.

Reply:
Great catch too! Indeed, these test data files should be amended accordingly too.

Reply:
Excellent, the respective files have been removed. Waiting on tests to pass.